### PR TITLE
Added build file for itsxpress 1.7.2

### PR DESCRIPTION
**Pull request by arivers**
* [x ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/itsxpress/1.7.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+about:
+  home: http://github.com/usda-ars-gbru/itsxpress
+  license: 'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication'
+  license_family: 'PUBLIC-DOMAIN'
+  license_file: ''
+  summary: 'ITSxpress: Software to rapidly trim the Internally Transcribed Spacer (ITS) region from FASTQ files'
+
+
+extra:
+  recipe-maintainers: 'Adam R. Rivers'
```
Comment (bgruening) [inline recipes/itsxpress/1.7.1/meta.yaml]: please add here your github username

File: recipes/itsxpress/1.7.1/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - itsxpress
+  commands:
+    - itsxpress --help
+
+about:
+  home: http://github.com/usda-ars-gbru/itsxpress
+  license: 'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication'
```
Comment (bgruening) [inline recipes/itsxpress/1.7.1/meta.yaml]: Can you just use CC0 1.0

File: recipes/itsxpress/1.7.1/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  license_family: 'PUBLIC-DOMAIN'
+  license_file: ''
```
Comment (bgruening) [inline recipes/itsxpress/1.7.1/meta.yaml]: LICENSE.txt

**Review by bgruening — COMMENTED** [review]
@arivers can you please add my comments to the newer recipe. Thanks!

File: recipes/itsxpress/1.7.1/meta.yaml (hunk 1 above)
Comment (arivers) [reply recipes/itsxpress/1.7.1/meta.yaml]: I wasn't quite sure the correct format but after looking at some other meta.yml examples I think this is correct:

```
 recipe-maintainers:
    - arivers
```

File: recipes/itsxpress/1.7.1/meta.yaml (hunk 2 above)
Comment (arivers) [reply recipes/itsxpress/1.7.1/meta.yaml]: Changed to 

```
license: 'CC0 1.0'
```

File: recipes/itsxpress/1.7.1/meta.yaml (hunk 3 above)
Comment (arivers) [reply recipes/itsxpress/1.7.1/meta.yaml]: Would it be possible to do this on the next update?

I had omitted LICENSE.txt from the MANIFEST.in file so it was not in the Python package distribution. My conda recipe builds from Pypi and I've already submitted v1.7.2 to it. To include the LICENCE.txt file I would have to bump the software version to 1.7.3.  I've aready made the suggested changes on github https://github.com/USDA-ARS-GBRU/itsxpress/commit/f89fc5775fea709ca9e7db0d319815ffeb9014d7 so it should be in there next time.

**Pull request merged**